modules/infra/datapath/control_input.c

```diff
@@ -74,6 +75,20 @@ control_input_process(struct rte_graph *graph, struct rte_node *node, void **, u
 	return n;
 }
 
+static int control_input_init(const struct rte_graph *graph, struct rte_node *node) {
+	node->ctx_ptr = gr_pktmbuf_pool_get(graph->socket, RTE_GRAPH_BURST_SIZE);
+
+	if (node->ctx_ptr == NULL)
+		ABORT("gr_pktmbuf_pool_get(control_input): %s", rte_strerror(rte_errno));
```

Comment:
`ABORT()` seems excessive here. Maybe just return -1 and log an error?

Reply:
Without this, all control path features (ARP requests, ... ) won't work: it's better to quit now than having a non working system, isn't it ?